lib/EnhancedSwitch.js

```diff
@@ -247,9 +247,11 @@ var EnhancedSwitch = (function (_React$Component) {
       }
     }
   }, {
-    key: 'isChecked',
-    value: function isChecked() {
-      return this.refs.checkbox.checked;
+    key: 'setIndeterminate',
+    value: function setIndeterminate() {
+      if (this.props.indeterminate) {
+        this.refs.checkbox.indetermiante = true;
```

Comment:
@dclowd9901 Good catch. This is what happens when you have to write `indeterminate` way too may times. Created a PR to fix it